tests/components/zwave/test_init.py

```diff
@@ -617,6 +618,63 @@ def mock_connect(receiver, signal, *args, **kwargs):
         node.node_id, value.instance, value.index, value.command_class
     )
 
+    @patch.object(zwave, "import_module")
+    @patch.object(zwave, "discovery")
```

Comment:
patch don't work with async tests. You need to do those inside the test:

```
with patch.object(zwave, "import_module") as import_module, patch.object(zwave, "discovery") as discovery:
    hass.start()
    registry = mock_registry(hass)
    …
```

Reply:
Also, this test is _inside_ another test? The indentation should be 0 spaces for the definition